mlir/include/mlir/Dialect/Linalg/IR/LinalgInterfaces.h

```diff
@@ -111,7 +111,8 @@ FailureOr<ConvolutionDimensions> inferConvolutionDims(LinalgOp linalgOp);
 
 /// Checks whether `linalgOp` conforms to ConvolutionOpInterface.
 // TODO: embed within `isa<ConvolutionOpInterface>` if possible / natural.
```

Comment:
Document the behavior of `allowEmptyConvolvedDims` in the comment here.

Reply:
> Document the behavior of `allowEmptyConvolvedDims` in the comment here.

Added~